ui/test/callWithTelemetryAndErrorHandling.test.ts

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
```

Comment:
Last time I tried doing this I have some issues because the vscode module wasn't defined, since this wasn't an actual extension. What did you do to solve this?

Reply:
I just created a super simple "test" extension. That's what these files are for:
ui/test/extension/extension.js
ui/test/extension/package.json

Reply:
And then I use the typical "launch.json" that we use in the extension repos, except I pass "test/extension" in as `CODE_EXTENSIONS_PATH`